kolibri/plugins/style_guide/templates/style_guide/style_guide.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Unfortunately, <mat-svg> seems to rely on a WebPack preloader called "svg-icon-inline-loader", which won't be able to work with the stringified code example we have in the style guide.

And I thought the style guide will be used internally (at least for now), so having external dependency shouldn't be a problem?